DataFormats/Detectors/ITSMFT/MFT/src/TrackMFT.cxx

```diff
@@ -52,6 +52,77 @@ void TrackMFT::setCovariances(const SMatrix55& covariances)
   mCovariances = covariances;
 }
 
+//_________________________________________________________________________________________________
+void TrackMFT::extrapHelixToZ(double zEnd, double Field)
+{
+  /// Track extrapolated to the plane at "zEnd" considering a helix
+
+  if (getZ() == zEnd) {
+    return; // nothing to be done if same z
+  }
+
+  // Extrapolate covariances
+  extrapHelixToZCov(zEnd, Field);
+
+  // Extrapolate track parameters
+  double dZ = (zEnd - getZ()); // Propagate in meters
+  double cosphi0 = TMath::Cos(getPhi());
+  double sinphi0 = TMath::Sin(getPhi());
```

Comment:
you can combine sin, cos call via https://github.com/AliceO2Group/AliceO2/blob/0955bd3ec8fcda223f197c2a62c9feb897d7cb97/Common/MathUtils/include/MathUtils/Utils.h#L70-L78

Reply:
Done